packages/by-name/kata-image/package.nix

```diff
@@ -5,11 +5,12 @@
 , stdenv
 , stdenvNoCC
 , distro ? "cbl-mariner"
+, microsoft
 , bubblewrap
 , fakeroot
 , fetchFromGitHub
 , fetchurl
-, kata-agent
+, kata-agent ? microsoft.kata-agent
```

Comment:
This doesn't really make sense, as it builds msft's kata-image and you can't just inject the upstream agent. Same for the kernel-uvm.

Reply:
What? This imports the microsoft agent? That's what it did before, right?

Reply:
It makes the kata-agent configurable, and let's it default to the microsoft.kata-agent. Instead, we can just always use microsoft.kata-agent (or just kata-agent, when it's in the same subset?).